### ISSUE TITLE
changes in yolov3.cfg

### ISSUE THREAD

**Issue by Anirudh-T-P** [qqwweee/keras-yolo3#133]
is there any changes to be made on yolov3.cfg for training it on our own dataset?

**Comment by ErrEss** [comment]
Even I have the same doubt. Are we supposed to change anything in yolov3.cfg?

**Comment by spate141** [comment]
why would you want to make changes in `yolov3.cfg` ? it's not used during training.

**Comment by sherlockchou86** [comment]
the yolov3.cfg is used when create the yolo_weights.h5, the yolo_weights.h5 is used for our training, right?

actually i have a question why we need load the pretrained weights.  new for machine learning, thanks!

**Comment by ErrEss** [comment]
@sherlockchou86 correct, that is the reason. Moreover I have been training the model by unfreezing most of the layers still the lost does not drop below 12. So I am suspecting there might be some changes to be made in yolo.cfg.

**Comment by sherlockchou86** [comment]
@ErrEss did your model work for your test even with loss 12.0 ?

**Comment by sherlockchou86** [comment]
my last val_loss is about 13.0, and it works fine for me!

**Comment by gittigxuy** [comment]
@sherlockchou86 ,when I train VOC data,I encounter some problem,could you please help me?
https://github.com/qqwweee/keras-yolo3/issues/136

**Comment by spate141** [comment]
@sherlockchou86: `yolov3.cfg` is the config file used to convert original yolo weights to Keras weights, which later is used in different parts of this repo; i.e. for train, predict.

About pre-trained weights: Please check [Why use pre-trained models?](https://www.learnopencv.com/keras-tutorial-using-pre-trained-imagenet-models/) or check any computer vision blog-tutorial mentioning usage of pre-trained weights.

**Comment by sherlockchou86** [comment]
@spate141  yes, so the yolov3.cfg file is useful for our training, right?   how to change the cfg file if i want to make the model more simple, improving the process speed when predicting.  thanks!

**Comment by spate141** [comment]
@sherlockchou86 You can't make the model more simple, improving the process speed when predicting by changing the config file!

**Comment by sherlockchou86** [comment]
@spate141  sorry  i mean if i need change the cfg and train it with darknet first，and then use the weight file to train in keras？need i modify my code in keras-yolov3？such as change the nn layers？
or do you know how to make the model more simple  in keras（total 100 layers ok for me）thanks！
